CodeGen/Generators/UnitsNetGen/QuantityGenerator.cs

```diff
@@ -364,36 +364,37 @@ private void GenerateStaticMethods()
         /// <param name=""unitConverter"">The <see cref=""UnitConverter""/> to register the default conversion functions in.</param>
         internal static void RegisterDefaultConversions(UnitConverter unitConverter)
         {{
-            // Register in unit converter: BaseUnit -> {_quantity.Name}Unit");
+            // Register in unit converter: {_quantity.Name}Unit -> BaseUnit");
```

Comment:
I reversed these to better match our "mental model" in json of going from a unit to base units first.